scripts/createTestPlugin.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { spawnPiped, waitFor, waitForExit, print } = require('./helpers');
+
+async function createTestPlugin() {
+  print('Creating a Backstage Plugin');
+  const createPlugin = spawnPiped(['yarn', 'create-plugin']);
```

Comment:
Not sure if we should run this in the newly created app or this repo 🤔, thoughts?

Reply:
It will probably speed up the time to run the e2e test if we'd combine the two?

Reply:
Yeah I think they should be combined, but wondering whether we would try create-plugin inside this repo or in the app that we just created. I'm thinking it makes sense to test it in the newly created app, since we'll want to run yarn start there anyway. Its also less of a disruption if plugin creation breaks in this repo compared to other ones.

I believe I just answered my own question, let's run the create-plugin inside the app that we create in the tmp folder ^_>